src/PrivateDns/PrivateDns.Test/ScenarioTests/ZoneTests.ps1

```diff
@@ -104,17 +102,17 @@ Zone CRUD with piping
 function Test-ZoneCrudWithPiping
 {
 	$zoneName = Get-RandomZoneName
-    $createdZone = TestSetup-CreateResourceGroup | New-AzPrivateDnsZone -Name $zoneName -Tags @{tag1="value1"}
-
-	$resourceGroupName = $createdZone.ResourceGroupName
+    $resourceGroup = TestSetup-CreateResourceGroup 
```

Comment:
Fix indentation

Reply:
At all places in this file, like L143

Reply:
not sure why it shows up like this here. the file has no issues